client/v3/client.go

```diff
@@ -68,7 +68,10 @@ type Client struct {
 	// Username is a user name for authentication.
 	Username string
 	// Password is a password for authentication.
-	Password        string
+	Password string
+	// Token is a JWT used for authentication instead of a password.
+	Token string
```

Comment:
We should add a validation on the config, only the following three cases are allowed,
- Only username & password are specified;
- Only token is specified;
- Neither is specified.

If both username & password and token are specified, then we should raise an error or warning.

Reply:
Added validation for this in the `New` path as well as a unit test.